protocol-designer/src/components/modals/CreateFileWizard/StagingAreaTile.tsx

```diff
@@ -0,0 +1,155 @@
+import * as React from 'react'
+import { useSelector } from 'react-redux'
+import without from 'lodash/without'
+import { i18n } from '../../../localization'
+import {
+  DIRECTION_COLUMN,
+  Flex,
+  Text,
+  SPACING,
+  JUSTIFY_SPACE_BETWEEN,
+  ALIGN_CENTER,
+  PrimaryButton,
+  DeckConfigurator,
+} from '@opentrons/components'
+import {
+  DeckConfiguration,
```

Comment:
I guess this can be import type { DeckConfiguration } from '@opentrons/shared-data'

```suggestion
  DeckConfiguration,
```